src/main/java/cloud/klasse/backendbusiness/teacher/TeacherService.java

```diff
@@ -0,0 +1,67 @@
+package cloud.klasse.backendbusiness.teacher;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class TeacherService {
+
+    private final TeacherRepository teacherRepository;
+
+    public Teacher createTeacher(final CreateTeacherModel model) {
+        Teacher teacher = teacherRepository.save(new Teacher(0, model.getUserName(), model.getNickName(), model.getEmail(),
```

Comment:
Teacher should not be validated on creation?

Reply:
Teachers should be varified manually

Reply:
Ok. I will fix it.